packages/console/src/console-history/ConsoleHistoryItem.scss

```diff
@@ -12,9 +12,31 @@ $button-vert-margin: ConsoleVariables.$button-vert-margin;
 }
 
 .console-history-item-command {
+  position: relative;
   white-space: pre-line;
 }
 
+.console-history-actions {
+  z-index: 1000;
```

Comment:
We shouldn't have the z-index so high. In bootstraps `_variables.scss`, they define a bunch of `$zindex-*` variables, and `$zindex-dropdown` is `1000`, so this could in theory conflict with that. Probably should only set it to 100 at most.
That being said - why do we need to set this anyway?